packages/office-ui-fabric-react/src/components/TextField/examples/TextField.AutoComplete.Example.tsx

```diff
@@ -1,17 +1,23 @@
 import * as React from 'react';
 import { PrimaryButton } from 'office-ui-fabric-react/lib/Button';
 import { TextField } from 'office-ui-fabric-react/lib/TextField';
+import { getId } from 'office-ui-fabric-react/lib/Utilities';
 import './TextField.Examples.scss';
 
 export class TextFieldAutoCompleteExample extends React.Component<any, any> {
+  private _textFieldId: string = getId('autoCompleteTextField');
+  private _labelId: string = getId('autoCompleteLabel');
+
   public render(): JSX.Element {
     return (
-      <form action="" className="docs-TextFieldExample">
-        <TextField
-          label="Fill in and submit this form. The page will reload and autocomplete suggestions will appear."
-          name="example"
-          autoComplete="on"
-        />
+      <form action="" className="docs-TextFieldExample" aria-labelledby={this._labelId}>
+        <div className="ms-TextFieldStandaloneLabel">
```

Comment:
Can we get rid of the wrapping div and change the label to display block if necessary? Also use a fabric label. :)

Reply:
Good suggestions @ecraig12345! It looks like we may able to get the Fabric `Label` for "free", see https://github.com/OfficeDev/office-ui-fabric-react/pull/8226/files#r263979188